app/Models/Product.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    protected $primaryKey = 'product_id';
+    protected $table = 'tbl_product';
+    
+    protected $fillable = [
+    	'brand_id','product_name','product_price','product_image'
+    ];
```

Comment:
indent 4 spaces, xuống dòng và thêm , vào phần tử cuối cho m, sửa các chỗ tương tự khác 
```suggestion
    protected $fillable = [
        'brand_id',
        'product_name',
        'product_price',
        'product_image',
    ];
```